### DISCUSSION TITLE
Can Nextjs preload css chunks?

### DISCUSSION THREAD

**Discussion by ArronHsiao** [vercel/next.js#17909] [Help]
Hi, I have import some css from 3rd parties in component, say import "A.css" and import "B.css" inside one component. After built I got /_next/static/css/**some_numbering.chunk.css**. But when I do PageSpeed, it tells me the **some_numbering.chunk.css** should load first to show the right style. Is there any way to preload the built css chunks ?

**Comment by nicosh** [comment]
@ArronHsiao any news on this?
